src/lib.rs

```diff
@@ -825,6 +828,13 @@ impl Config {
         self.dgram_recv_max_queue_len = recv_queue_len;
         self.dgram_send_max_queue_len = send_queue_len;
     }
+
+    /// Configures whether to enable Packet Pacing.
+    ///
+    /// The default is `true`.
+    pub fn enable_pacing(&mut self, v: bool) {
```

Comment:
Why do we need a configuration for this? The application can decide to ignore the timestamp value (or just not call `send_at()` at all) even if this is enabled by default.

Reply:
Sure, this is not needed clearly.